cvss_util/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.2"
+__version__ = "2.3.0"
```

Comment:
Hmm we can probably delete this now - as we aer using pbr.

Reply:
I get this upon running setup.py once removed: `package init file 'cvss_util/__init__.py' not found (or not a regular file)` – what would you do here?

Reply:
Something like this?

```
import pbr.version


__version__ = pbr.version.VersionInfo(
    'cvss-util').version_string()
```